### ISSUE TITLE
Trapped in the Nether with no Portal

### ISSUE THREAD

**Issue by SaintCeadda** [Darkosto/SevTech-Ages#2169]
Did an issue search and didn't find a similar issue. I have been playing on a server and just reached Age 3. Traveled to a desert and found one of the giant Nether portals. Activated it and stepped through.

The server timed out and everyone was kicked off. This is normal behavior each time we activate a new dimension the first time.

However, after rejoining, I had spawned in the Nether in a wall. Broke the netherrack to escape and looked all around, but cannot find the return portal. It didn't generate one. Now I'm trapped in the Nether wondering where my return portal is. Is this a common issue?

**Comment by thephoenixlodge** [comment]
the server may have put you in the nether at the coords that you were in the overworld, whereas the portal would've genned at the transformed coords (overworld coords divided by 8). If you can get the coords of the portal in the overworld, (ask someone to go visit it for you?) you should be able to figure out roughly where the portal should've spawned.

**Comment by SaintCeadda** [comment]
I'll give that a try, thanks.

**Comment by Jandakast** [comment]
This is an issue with the server crashing while switching/generating the dimension. This happens in vanilla as well. Not a pack issue.

You can try using /gchouston or having the OP TP you home

**Comment by kevinsh7** [comment]
It's like I'm playing a different game. I'm on version 3.0.6 single player. The lesser shoggoth don't spit acid for me - was this disabled then? The first two dens I found, I backed off a distance, build a "hidey hole" for the night, cached most of my gear and went to sleep. I then found the first lair empty, so I harvested all the monolith stone and figured that I could always trade with farmers for the shoggoth flesh if I really needed to.  Then I found a second lair and confirmed it was alive and kicking. I backed off, created another "safe house", slept for the night, only to return to a dead lair - but it wasn't entirely empty. There were two frozen lesser shoggoth in the den. I don't even think they had collision shells, but maybe I could only walk through parts of them. So I restarted the game and they disappeared. So sleeping near a den will definitely kill them all off.

The third den I found - no sleeping! - was out in a river. I stayed in the water, they don't swim well, so I was able to kill a few. Then I rushed up to the entrance and placed down a "fence" of monolith blocks in columns on every other tile across the entrance. Then I learned they can climb. lol. So I finished caging them in and was able to farm as many as I needed.

So perhaps I have a different bug? In any case, does anyone know if any of the other dimensions have different distance ratios, like the 8 to 1 ratio of the nether? If so, that could really be useful. I'm in Age 2 now and need a blue middle gem for the furnace controller. Having a hard time finding swamps with druids, so being able to use the hunting dimension, or The Beneath for a portal network would really help with the exploration (assuming I can survive the plentiful mobs.)
